engine/order_manager.go

```diff
@@ -857,11 +841,19 @@ func (m *OrderManager) UpsertOrder(od *order.Detail) (resp *OrderUpsertResponse,
 	return upsertResponse, nil
 }
 
-// get returns all orders for all exchanges
-// should not be exported as it can have large impact if used improperly
+// get returns all orders for all exchanges should not be exported as it can
+// have large impact if used improperly
```

Comment:
I think this comment can be changed now that you're copying instead. `Get returns a copy of all orders`